registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -92,6 +92,7 @@
     "sha256:bf8b7c3ed427086138e673268edc720407bd8cb7caf2e2aa3a0e3d68277740e5": ["v1.10.4"]
     "sha256:d93407d031296336ccbabc8494005672dc048c4ebc616ccfc18f813d49bd87fc": ["v1.10.5"]
     "sha256:46254acdd8a372b938c80a73fbc87c4473a4d6c142a97ab76728eae5d493171a": ["v1.10.6"]
+    "sha256:3cda2bb5f363f60eb5a6b6d84d047c2afd1cce547e6b4fea445d590fee39779d": ["v1.10.7"]
```

Comment:
SHA `3cda2bb5f363` matches cluster-api-controller v1.10.7 in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/cluster-api-controller@sha256:3cda2bb5f363f60eb5a6b6d84d047c2afd1cce547e6b4fea445d590fee39779d/details).